posixlib/src/main/resources/scala-native/unistd.c

```diff
@@ -11,16 +11,6 @@ extern char **environ;
 extern char *optarg;
 extern int opterr, optind, optopt;
```

Comment:
If these are required in the glue code then I think we should add some documentation to clarify. I think since that are defined in POSIX the same we may not need them especially since then are not referenced via `@name` now.

Reply:
You are right. Will remove in https://github.com/scala-native/scala-native/pull/3048

Reply:
https://github.com/scala-native/scala-native/pull/3048/files#diff-9714e0fa969c14c825759d243ec79185ff73a9d5dea6e9309a7d48f9ee7750eaL11

Reply:
Housekeeping....

I am pretty sure than lines 11 & 12 could & should be deleted.

Unless my blurry & beaten eyes are missing something, they are not used in that .c file.

They say "somewhere else somebody is actually allocating space for 'optarg' & friends." not
"please allocate space here".  I believe the space is allocated in glibc ("-libc").

I do not know why I have never twigged to removing those before, I've spent enough time
in that file.

They do no harm, just add a bit of "Why are these here?" to anyone reading the code.